app/Models/Loggable.php

```diff
@@ -144,7 +144,9 @@ public function logCheckin($target, $note)
 
         $log->location_id = null;
         $log->note = $note;
-        $log->user_id = Auth::user()->id;
+        if(Auth::user())
+            $log->user_id = Auth::user()->id;
```

Comment:
Style nit - I prefer if statements with `{}` - not having them can lead to extremely obscure bugs.

Is there a way we can specify some kind of Admin user here instead - just so we don't have a `null` for the `user_id`. Maybe that's outside of the scope of this PR.

Reply:
> Style nit - I prefer if statements with {}

Fixed.

> Is there a way we can specify some kind of Admin user here instead 

It's outside the scope of the PR, IMHO. We could force it via cli argument, but it could be forged anyway, so if you run stuff via cli, that's just kinda the way it goes :(